src/utilities/Constants.ts

```diff
@@ -0,0 +1,30 @@
+export const PageRoutes = {
```

Comment:
Not sure about these in here. Might have a separate folder within utilities called Constants and then individual files for these in there

Reply:
sounds good to me - @hazel-davis @NogaAD your thoughts?

Reply:
sounds good to me - @hazel-davis @NogaAD your thoughts?